pkg/controller/gc_controller.go

```diff
@@ -115,6 +115,12 @@ func (c *gcController) cleanBackups() {
 			}
 
 			for _, volumeBackup := range backup.Status.VolumeBackups {
+				if c.snapshotService == nil {
+					glog.Warningf("Cannot garbage-collect snapshot %s for %backup s/%s because server is not configured with PersistentVolumeProvider",
```

Comment:
typo: `%backup s/%s`

If the snapshot service is nil, the net result is that backups will be GC'd but associated snapshots will be orphaned?